elections57_results.py

```diff
@@ -223,16 +225,19 @@ def get_class(v):
     from datetime import datetime
 
     def parse_timestamp(s):
-        return datetime.strptime(s, '%m/%d/%Y %H:%M:%S')
+        return datetime.strptime(s, '%Y-%m-%d %H:%M:%S')
+
+    # See http://stackoverflow.com/questions/6832445/how-can-bcrypt-have-built-in-salts
+    salt = bcrypt.gensalt()
 
     # Dump in vote order, i.e. sort by timestamp.
     for d in sorted(data.items(), key=lambda x: parse_timestamp(x[1][col_timestamp])):
         print u'{}\t{}\t{}'.format(
-                d[0], # TODO: safely hash it
+                bcrypt.hashpw(d[0] + cmdline_args.secret, salt), 
                 # Dump it in ISO format instead of C one used in raw data.
                 parse_timestamp(d[1][col_timestamp]),
                 # Omit columns containing information identifying the voter.
-                '\t'.join(d[1][col_sub_news:col_comment])
+                '\t'.join(d[1][col_candidates:col_bylaws])
```

Comment:
We don't need a range here, nor `join()`, just `d[col_candidates]` would be simpler and more clear.